Services/CVService/Migrations/20201012190715_InitialCreate.Designer.cs

```diff
@@ -5,12 +5,12 @@
 using Microsoft.EntityFrameworkCore.Metadata;
 using Microsoft.EntityFrameworkCore.Migrations;
 using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
-using Studfolio.CompanyService.Database;
+using Studfolio.CVService.Database;
```

Comment:
Bad. You can't change files from another services in the Pull Request connected with the CVService. Of course, you can lend someone's code, but 1) not the migrations code and 2) don't steal their files (now I don't see the migration from CompanyService cause that file is there)